include/dlaf/factorization/qr/t_factor_mc.h

```diff
@@ -17,12 +17,14 @@
 
 #include "dlaf/factorization/qr/api.h"
 
+#include "dlaf/common/assert.h"
 #include "dlaf/common/data.h"
 #include "dlaf/common/index2d.h"
 #include "dlaf/common/pipeline.h"
 #include "dlaf/common/range2d.h"
 #include "dlaf/common/vector.h"
 #include "dlaf/communication/functions_sync.h"
+#include "dlaf/lapack/enum_output.h"
```

Comment:
without this line it will not compile, I think it is needed because here:
https://github.com/eth-cscs/DLA-Future/blob/dd819b8a5bf1409396bbd3c0ec5c0c15a65277ca/include/dlaf/lapack/enum_output.h#L41
we use `lapack::matrixtype2str`. 
But I haven't found out how to avoid this `#include`: any suggestion?

Reply:
But you are not including it where you are actually using it…

Reply:
you're perfectly right..